assets/js/members.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Parse URL parameter
+ *
+ * @param name {string} Parameter name
+ * @param url {string} URL to be parsed
+ * @returns {string|null} Parameter value
+ */
+const getParameterByName = (name, url) => {
+  const urlToParse = url || window.location.href;
+  const nameToParse = name.replace(/[[\]]/g, '\\$&');
+  const regex = new RegExp(`[?&]${nameToParse}(=([^&#]*)|&|#|$)`);
+  const results = regex.exec(urlToParse);
+
+  if (!results) {
+    return null;
+  }
+
+  if (!results[2]) {
+    return '';
+  }
+
+  return decodeURIComponent(results[2].replace(/\+/g, ' '));
+};
+
+const toggleContent = (triggerEl) => {
+  const targetEl = document.querySelector(triggerEl.getAttribute('data-toggle'));
+  const isTargetVisible = !targetEl.hasAttribute('hidden');
+  triggerEl.setAttribute('aria-expanded', !isTargetVisible);
+  targetEl.toggleAttribute('hidden');
+};
+
+// Handle toggle elements.
+const toggleElements = document.querySelectorAll('[data-toggle]');
+
+if (toggleElements.length) {
+  toggleElements.forEach((element) => {
+    element.addEventListener('click', (event) => toggleContent(event.currentTarget));
+  });
+}
```

Comment:
1) Za sebe bych třeba název atributu (`is-hidden`) dala do proměnné, když jej používáš i na řádku 29.

2) Před řádkem 27 bych raději dala:
```
if (targetEl) {
...
}
```

3) Řádek 35 bych zahodila, ale to už víš proč 🙂  
(https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll)

Reply:
1. Je to standardní atribut, ničím jiným se obsah, pokud vím, v HTML skrýt nedá… Kdyby to byla CSS třída nebo data atribut, tak ano, ale nativní atribut mi přijde v pohodě do proměnné nedávat.